src/share/routes.js

```diff
@@ -0,0 +1,31 @@
+import { format404 } from '../lib/html-formatter';
+
+const importRegex = /\/import(\/*)/;
+const exportRegex = /\/export(\/*)/;
+const shareRegex = /\/share(\/?)$/;
+//open package.json post build-server
+export default (workbox, Share) => {
+    workbox.routing.registerRoute(
```

Comment:
We don't need any of this, we'll just use the HTML files directly.

Reply:
You can remove this entire file.  The only thing you might want to do in here is to add some dynamic runtime caching for the files you are serving (e.g., `import.html`, `import.js`, etc) so it will work offline.  Same thing we did with the docs routes.